pkg/manifests/rte/rte.go

```diff
@@ -70,6 +73,8 @@ type UpdateOptions struct {
 
 func (mf Manifests) Update(options UpdateOptions) Manifests {
 	ret := mf.Clone()
```

Comment:
IMHO it's the bug in the `mf.Clone` method, it should just deep copy all entries under the initial struct